src/main/java/classify/commands/EditStudent.java

```diff
@@ -16,13 +16,13 @@
 
 //@@author blackmirag3
 public class EditStudent extends Commands {
-    private static final String ONE = "1";
-    private static final String TWO = "2";
-    private static final String THREE = "3";
-    private static final String FOUR = "4";
-    private static final String FIVE = "5";
-    private static final String SIX = "6";
-    private static final String SEVEN = "7";
+    private static final String ADD_SUBJECT = "1";
+    private static final String EDIT_SUBJECT = "2";
+    private static final String DELETE_SUBJECT = "3";
+    private static final String EDIT_NUMBER = "4";
+    private static final String EDIT_REMARKS = "5";
+    private static final String EDIT_PAYMENT_DATE = "6";
+    private static final String EDIT_GENDER = "7";
```

Comment:
Good nomenclature so that the developer reading the code can understand these magic numbers. 

![heres-an-image-of-drip-bocchi-holding-up-a-blank-sheet-of-v0-8h1fkr02kmqa1.jpeg](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/144fe84d-ecc9-47aa-b9a7-923a82ca82b9)